standard/coordinates.md

```diff
@@ -0,0 +1,412 @@
+as the element's `exit`. If the element is reversed, the `entrance` and `exit` ends are switched.
+That is, for a `reversed` element, particles traveling downstream will
+enter at the element's `exit` end and will exit at the `entrance` end.
+
+The procedure to connect elements together to form the branch coordinates is to ignore 
+`element body` alignment shifts and mate the `downstream` reference frame of
+the element with the `upstream` reference frame of the next element in
+the branch so that the {math}`(x,y,z)` coordinate axes coincide.
+If there are body alignment shifts, the `entrance` and `exit` frames will move with the element. 
+However, this does not affect the branch coordinate system.
+This is illustrated in [](#f:patch.between). [](#f:patch.between)A shows the general situation
+with the downstream frame of element `A` mated to the upstream frame of element `B`.
+The {math}`(x,z)` coordinates are drawn at the entrance end of the elements and {math}`z` will 
+always point towards the element's exit end.
+[](#f:patch.between)B shows a line
+with an unreversed drift named `dft` connected to an unreversed bend named
+`bnd`. [](#f:patch.between)C shows the same line but with `bnd` reversed. This is like
+This gives an unphysical situation since a
+particle traveling through `dft` will "fall off" when it gets to the drift's end.
+[](#f:patch.between)D shows the same line as in [](#f:patch.between)C with the addition
+of a [`reflection patch`](#s:reflect.patch) `P` between `dft` and `bnd` to give a plausible geometry. 
+In this case, the patch rotates the coordinate system around the {math}`y`-axis by 180{math}`^o` 
+(in this example leaving the {math}`y`-axis invariant). This illustrates why
+a reflection patch is always needed between reversed and unreversed elements.
+
+Notes:
+- Irrespective of whether elements are reversed or not, the branch {math}`(x,y,z)` coordinate system
+at all {math}`s`-positions will always be a right-handed coordinate system.
+
+- Care must be take when using reversed elements. For example, if the field of the `bnd` element in
+the above example is appropriate for, say, electrons, that is, electrons will be bent in a clockwise
+fashion going through `bnd`, an electron going in a forward direction through the
+line in [](#f:patch.between)D will be lost in the bend
+(the {math}`y`-axis and hence the field is in the same direction for both cases so electrons 
+will still be bent in a clockwise fashion but with [](#f:patch.between)D a particle needs to be 
+bent counterclockwise to get through the bend). 
+That is, to get a particle going forward through the bend in [](#f:patch.between)D, positrons must be used.
+
+- A `reflection Patch` that rotated the coordinates, for example, 
+around the {math}`x`-axis by 180{math}`^o`  would also produce a plausible geometry.
+\end{itemize}
+
+%---------------------------------------------------------------------------------------------------
```

Comment:
It would probably be good to have a MAD expert look over the sections that follow.

Reply:
@cemitch99 OK if you know of anybody please feel free to ask them to look at it.